src/ORM/Query.php

```diff
@@ -1010,6 +1010,22 @@ public function enableHydration($enable = true)
         return $this;
     }
 
+    /**
+     * Disable hydrating entities.
+     *
+     * Disabling hydration will cause array results to be returned for the query
+     * instead of entities.
+     *
+     * @return $this
+     */
+    public function disableHydration()
+    {
+        $this->_dirty();
```

Comment:
Instead of duplicating the logic this must call the actual wrapper internally to prevent unDRY and copy paste bugs.

Reply:
I would rather have a couple of line of code duplication then adding additional method call.

Reply:
Dont nano opt a syntactic sugar wrapper. Thats a nogo, the wrapping is cheap and will prevent future bugs.